src/configs/passport.ts

```diff
@@ -0,0 +1,66 @@
+import { Request } from 'express';
```

Comment:
I have some doubts about placement of this file and about folder name and file itself.
What you think about moving all files from here to services, and rename at least this file from passport to `auth-service`? (logger is selfdescribing name and doesnt need it)

Reply:
fixed